app/scripts/modules/utils/EntryLog.js

```diff
@@ -0,0 +1,149 @@
+const ODataNode = require('./ODataNode.js');
+const multipartmixed2har = require('./multipartmixed2har.js');
+const formatXML = require('prettify-xml');
+
+function EntryLog () {
```

Comment:
minor: to me the name "EntriesLog" is a bit more understandable, as "Entry" suggest a single entry, so I was first mislead before looking into more detail to the actual code